src/Components/Login/Login.test.js

```diff
@@ -1,15 +1,97 @@
 import React from 'react';
-import Login from './Login';
+import { Login, mapState, mapDispatch } from './Login';
 import { shallow } from 'enzyme';
+import { currentUser, loggedIn } from '../../actions/actions';
+import { getUser } from '../../apiCalls';
 
 describe('Login', () => {
-  let wrapper
+  
+  let wrapper, mockEvent, mockGetUser
 
-  beforeEach(() => {
-    wrapper = shallow(<Login />)
-  })
+  describe('login Component', () => {
+
+    beforeEach(() => {
+      mockEvent = { target: {name: 'email', value:'abc123@aol.com'} }
+      mockGetUser = jest.fn()
+      wrapper = shallow(<Login />)
+    });
+
+    it('Should match the snapshot', () => {
+      expect(wrapper).toMatchSnapshot()
+    });
+
+    it('Should setState when handleChange is called', () => {
+      wrapper.setState({ email: '', password:'', error: '' });
+      wrapper.instance().handleChange(mockEvent);
+  
+      expect(wrapper.state()).toEqual({ email: 'abc123@aol.com', password:'', error: '' })
+    });
+
+    it('Should call handleChange when email input is changed', () => {
+      wrapper.instance().handleChange = jest.fn();
+      const mockEmailEvent = { target: {email: 'email', value:'abc123@aol.com'} }
+
+      wrapper.find('.login-input-email').simulate('change', mockEmailEvent);
+
+      expect(wrapper.instance().handleChange).toHaveBeenCalledWith(mockEmailEvent)
+    });
+
+    it('Should call handleChange when password input is changed', () => {
+      wrapper.instance().handleChange = jest.fn();
+      const mockPasswordEvent = { target: {password: 'password', value:'password123'} }
+
+      wrapper.find('.login-input-password').simulate('change', mockPasswordEvent);
+
+      expect(wrapper.instance().handleChange).toHaveBeenCalledWith(mockPasswordEvent)
+    });
+
+    it('Should call getUser with an email and password when handleLogin is called', () => {
+      const mockEvent = { preventDefault: jest.fn() };
+
+      wrapper.setState({ email: 'h', password: 'h2', error: '' });
+    
+      wrapper.find('button').simulate('click', mockEvent);
+
+      expect(getUser).toHaveBeenCalledWith(wrapper.state.email, wrapper.state.password)
+    });
```

Comment:
Down to work on this more with you this weekend.  Seems like if we get this one to pass, we'll be close to 100% coverage.